.github/workflows/conformance-ginkgo.yaml

```diff
@@ -276,9 +276,7 @@ jobs:
         with:
```

Comment:
@viktor-kurchenko can you drop the commit 3ea184f94df86a302ed48af421fef49447febba7 it's causing the CI to fail. I have to fix this on main so that it can be "backportable".

Reply:
Thank you!